README.md

```diff
@@ -0,0 +1,52 @@
+MIES is a sweep based data acquisition software package for intracellular
+electrophysiology (patch clamp). It offers top of its class flexibility and
+robustness for stimulus generation, data acquisition/evaluation and has
+proven to be a valuable tool in the literature \[[1](https://doi.org/10.1038/s41593-019-0417-0),
+[2](https://elifesciences.org/articles/37349)\].
```

Comment:
MIES is a proven [1, 2] sweep based data acquisition software package for intracellular electrophysiology (patch clamp). It offers top of its class flexibility and robustness for stimulus generation, data acquisition, and analysis.